deployments/edx/config/common.yaml

```diff
@@ -1,9 +1,3 @@
-nfsPVC:
-  enabled: true
-  nfs:
-    serverIP: 10.128.0.62
-    shareName: export/data8xv2homes-2022-02-16
-
 jupyterhub:
```

Comment:
you should add a common NFS block at the top of this file containing:

```
nfsPVC:
  enabled: true
  nfs:
    serverIP: 10.1280.62
```

Reply:
done good deal